themes/digital.gov/layouts/partials/core/footer.html

```diff
@@ -2,7 +2,11 @@
   <div class="grid-container grid-container-desktop usa-footer__return-to-top">
     <div class="grid-row">
       <div role="navigation" class="grid-col-auto">
-        <a href="#"><span>Return to top</span> <i class="fas fa-arrow-circle-up"></i></a>
+        <a href="#"><span>Return to top</span>
+          <svg class="dg-icon" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
Is this a custom icon?

Reply:
This is an override style and have noted this in `_icons.scss`.